sub-packages/bionemo-geometric/src/bionemo/geometric/atom_featurizers.py

```diff
@@ -0,0 +1,398 @@
+        return MAX_CHIRAL_TYPES
+
+    def get_features(self, atom: Atom) -> list[bool]:
+        """Returns features of the atom."""
+        return one_hot_enc(int(atom.GetChiralTag()), MAX_CHIRAL_TYPES)
+
+
+class TotalNumHFeaturizer(BaseFeaturizer):
+    """Class for featurizing atom by total number of hydrogens."""
+
+    def __init__(self) -> None:
+        """Initializes TotalNumHFeaturizer class."""
+        pass
+
+    @property
+    def n_dim(self) -> int:
+        """Returns dimensionality of the computed features."""
+        return MAX_NUM_HS
+
+    def get_features(self, atom: Atom) -> list[bool]:
+        """Returns features of the atom."""
+        return one_hot_enc(atom.GetTotalNumHs(), self.n_dim)
+
+
+class HybridizationFeaturizer(BaseFeaturizer):
+    """Class for featurizing atom by its hybridization type."""
+
+    def __init__(self) -> None:
+        """Initializes HybridizationFeaturizer class."""
+        pass
+
+    @property
+    def n_dim(self) -> int:
+        """Returns dimensionality of the computed features."""
+        return MAX_HYBRIDIZATION_TYPES
+
+    def get_features(self, atom: Atom) -> list[bool]:
+        """Returns features of the atom."""
+        return one_hot_enc(int(atom.GetHybridization()), self.n_dim)
+
+
+class AromaticityFeaturizer(BaseFeaturizer):
+    """Class for featurizing atom based on its aromaticity."""
+
+    def __init__(self) -> None:
+        """Initializes AromaticityFeaturizer class."""
+        pass
+
+    @property
+    def n_dim(self) -> int:
+        """Returns dimensionality of the computed features."""
+        return 1
+
+    def get_features(self, atom: Atom) -> list[bool]:
+        """Returns features of the atom."""
+        return [atom.GetIsAromatic()]
+
+
+class PeriodicTableFeaturizer(BaseFeaturizer):
+    """Class for featurizing atom by its position (period and group) in the periodic table."""
+
+    def __init__(self) -> None:
+        """Initializes PeriodicTableFeaturizer class."""
+        self.pt = Chem.GetPeriodicTable()
+
+    @property
+    def n_dim(self) -> int:
+        """Returns dimensionality of the computed features."""
+        return 25
+
+    def get_period(self, atom: Chem.Atom) -> list[bool]:
+        """Returns one-hot encoded period of atom."""
+        atomic_number = atom.GetAtomicNum()
+        # The number of elements per period in the periodic table
+        period_limits = [2, 10, 18, 36, 54, 86, 118]
+
+        # Determine the period based on atomic number
+        for period, limit in enumerate(period_limits, start=1):
+            if atomic_number <= limit:
+                return one_hot_enc(period - 1, 7)
+        return None
+
+    def get_group(self, atom: Chem.Atom) -> list[bool]:
+        """Returns one-hot encoded group of atom."""
+        group = self.pt.GetNOuterElecs(atom.GetAtomicNum())
+        return one_hot_enc(group - 1, 15)
+
+    def get_atomic_radii(self, atom: Chem.Atom) -> list[float]:
```

Comment:
better to add an option for normalization, either in the method call, or in the init

Reply:
same for other featurizers

Reply:
All features have been normalized to lie between 0 and 1. For example:https://github.com/NVIDIA/bionemo-framework/blob/sveccham/impl_atom_bond_feats/sub-packages/bionemo-geometric/src/bionemo/geometric/atom_featurizers.py#L204
Do you want me to expose a function to the user so that they can choose between normalized or unnormalized values?